influxdb/src/main/scala/akka/stream/alpakka/influxdb/scaladsl/InfluxDbFlow.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.influxdb.{impl, InfluxDbWriteMessage, InfluxDbWriteResult, InfluxDbWriteSettings}
+import akka.stream.scaladsl.Flow
+import org.influxdb.InfluxDB
+import org.influxdb.dto.Point
+
+import scala.collection.immutable
+
+/**
+ * Scala API to create InfluxDB flows.
+ */
+object InfluxDbFlow {
+
+  def create(settings: InfluxDbWriteSettings)(
+      implicit influxDB: InfluxDB
+  ): Flow[InfluxDbWriteMessage[Point, NotUsed], InfluxDbWriteResult[Point, NotUsed], NotUsed] =
+    Flow[InfluxDbWriteMessage[Point, NotUsed]]
+      .batch(settings.batchSize, immutable.Seq(_))(_ :+ _)
```

Comment:
I think we should consider leaving the grouping to the user instead of always using `batch`. `batch` will only group if there is back-pressure, but there might be benefits of grouping even when writing isn't slow eg. with [`groupedWeightedWithin`](https://doc.akka.io/docs/akka/current/stream/operators/Source-or-Flow/groupedWeightedWithin.html). I admit that the API becomes a bit cruder, showing batching in the examples should balance that, though.